XenAdmin/Wizards/NewPolicyWizard/NewPolicySnapshotTypePage.cs

```diff
@@ -43,20 +43,14 @@ namespace XenAdmin.Wizards.NewPolicyWizard
 {
     public partial class NewPolicySnapshotTypePage : XenTabPage, IEditPage
     {
-        protected List<VM> _selectedVMs;
-        public List<VM> SelectedVMs
-        {
-            get { return _selectedVMs; }
-            set
-            {
-                _selectedVMs = value;
-            }
-        }
+        private VMSS _policy;
+
+        public List<VM> SelectedVMs { private get; set; }
         
         public NewPolicySnapshotTypePage()
         {
             InitializeComponent();
-            this.labelWarning.Text = string.Format(this.labelWarning.Text, Messages.VMSS_TYPE);
+            labelWarning.Text = string.Format(labelWarning.Text, Messages.VMSS_TYPE);

```

Comment:
I think that `string.Format` is not needed here anymore, as we always use the same string (Messages.VMSS_TYPE) as the parameter, so we can included in the label's text directly.

Reply:
I've rebased the whole PR but only the third commit is new.